changelog.json

```diff
@@ -1,6 +1,14 @@
 {
   "repository": "newrelic/newrelic-node-apollo-server-plugin",
   "entries": [
+    {
+      "version": "5.1.0",
+      "changes": {
+        "security": [],
+        "bugfixes": [],
+        "features": []
+      }
+    },
```

Comment:
Should this be edited to classify the restructuring as a "feature"?

Reply:
No. It was a code refactor

Reply:
You could make a case we can just let this sit as it doesn't change any behavior